gradle/dependencies.gradle

```diff
@@ -75,6 +75,7 @@ versions += [
   jmh: "1.32",
   hamcrest: "2.2",
   log4j: "1.2.17",
+  log4j2: "2.17.0",
```

Comment:
Can this be bumped to `2.17.1` (https://logging.apache.org/log4j/2.x/changes-report.html#a2.17.1)?

Reply:
Yes, I did [LOG4J2-3256](https://issues.apache.org/jira/browse/LOG4J2-3256) for [KAFKA-12399](https://github.com/apache/kafka/pull/10244) and it will be also applied to this PR. :+1: